backend/run/copr-backend-resultdir-cleaner

```diff
@@ -162,7 +166,9 @@ def clean_in(resultdir, dry_run=True):
 
 
 def _main():
-    args = parser.parse_args()
+    logging.basicConfig(level=logging.DEBUG)
+    setup_script_logger(LOG, "/var/log/copr-backend/resultdir-cleaner.log")
```

Comment:
Can we do `os.path.join(opts["log_dir"], "resultdir-cleaner.log")` or you intentionally wanted to setup the logger before parsing the config?

Reply:
I just moved the lines (c&p), but using opts seems possible. PTAL.